components/StepsProgress.js

```diff
@@ -81,6 +81,106 @@ const SeparatorLine = styled(Box)`
     `}
 `;
 
+const StepMobile = styled(Flex)`
+  width: 100%;
+  align-items: center;
+`;
+
+const StepsOuter = styled(Flex)`
+  background: #f5f7fa;
+  padding: 16px;
+`;
+
+const StepsMobileLeft = styled(Box)`
+  flex-grow: 2;
+  flex-direction: column;
+`;
+
+const StepsMobileRight = styled(Flex)`
+  margin-left: auto;
+  width: 48px;
+  height: 48px;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  position: relative;
+`;
+
+const PieProgressWrapper = styled(Box)`
```

Comment:
If we don't use the additional props from components, for example `Box` here (`mx`, `width`...etc), then it's better to use a basic HTML tag - like `styled.div` instead to not add unnecessary classes.